provisioning/transport/amqp/src/SaslTpmHandler.cs

```diff
@@ -267,7 +264,7 @@ private enum SaslChallengeAction
         {
             First,
             Interim,
-            Final
+            Final,
```

Comment:
I don't expect this enum to grow any further, for what it's worth.

Reply:
Yeah, I just style them this way as a rule, for consistency.